stencil/src/components/resource-links/resource-links.scss

```diff
@@ -0,0 +1,20 @@
+pulumi-resource-links{
+    .container {
+        margin-bottom: 24px;
+    }
+    .heading {
+        color: #2e3872;
+        font-weight: 600;
+        font-size: 14px;
+        padding-bottom: 2px;
+    }
+    .links {
+        padding-left: 10px;
+        padding-top: 8px;
+        font-size: 14px;
+        color: #2e3872;
+        a:hover {
+            color: #414ebc;
```

Comment:
I cannot seem to locate where these colors exist. I got them by inspecting the elements in the TOC. I do not see anywhere where they are defined though, which is why I hard coded them.